src/librustc_parse/lib.rs

```diff
@@ -271,12 +267,8 @@ pub fn stream_to_parser<'a>(
 /// The main usage of this function is outside of rustc, for those who uses
 /// librustc_ast as a library. Please do not remove this function while refactoring
 /// just because it is not used in rustc codebase!
-pub fn stream_to_parser_with_base_dir<'a>(
-    sess: &'a ParseSess,
-    stream: TokenStream,
-    base_dir: Directory,
-) -> Parser<'a> {
-    Parser::new(sess, stream, Some(base_dir), true, false, None)
+pub fn stream_to_parser_with_base_dir<'a>(sess: &'a ParseSess, stream: TokenStream) -> Parser<'a> {
```

Comment:
Hmm, there's no `base_dir` anymore, so perhaps the whole function is no longer needed.

Reply:
Yeah let's remove it. Tools can use `stream_to_parser` instead.